doc/api/axes_api.rst

```diff
@@ -14,18 +18,15 @@
    :no-members:
    :no-undoc-members:
 
-Inheritance
-===========
-.. inheritance-diagram:: matplotlib.axes.Axes
-   :private-bases:
-
 The Axes class
 ==============
 
-.. autoclass:: Axes
```

Comment:
The problem seems to be that `autosummary` is also providing toc entries for all of the methods hence the many duplicates.

It does not look like we can paramaterize auto summary to pass `:no-members:`.  Maybe the best solution is to push this to the bottom of the page?

Reply:
We can. We only need a custom autosummary template for that. See updated commit.